website/src/app.css

```
@@ -0,0 +1,483 @@
+@import url('https://api.fontshare.com/v2/css?f[]=cabinet-grotesk@400,500,700,800&display=swap');
```

Stylelint (17.7.0)

[error] Expected "url('https://api.fontshare.com/v2/css?f[]=cabinet-grotesk@400,500,700,800&display=swap')" to be "'https://api.fontshare.com/v2/css?f[]=cabinet-grotesk@400,500,700,800&display=swap'" (import-notation)

(import-notation)